### PR TITLE
Playzone

### PR DESCRIPTION
**Pull request by Smiranin**
Add playzone entity to the game. It covers game process algorithm and supports single and multiplayer.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "vsicons.presets.angular": true
```
Comment (stevermeister) [inline .vscode/settings.json]: do we need this in PR?

File: src/app/playzone/game-play.service.ts [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+  public endGame() {
+    this.destroy();
+    this._dbService.updateStateOnFireBase(this._roomId, this._cards, [], this._users, this.countHiddenBlock)
+      .then(() => this._router.navigate([`playzone/${this._roomId}/result`]));
+
+  }
+
+
+  public destroy() {
```
Comment (stevermeister) [inline src/app/playzone/game-play.service.ts]: destroy?
destroy what?

File: .vscode/settings.json (hunk 1 above)
Comment (Smiranin) [inline .vscode/settings.json]: Deleted .vscode folder.

File: src/app/playzone/game-play.service.ts (hunk 2 above)
Comment (Smiranin) [inline src/app/playzone/game-play.service.ts]: Renamed.

**Pull request merged**